js/views/modals/editListing/EditListing.js

```diff
@@ -750,6 +750,7 @@ export default class extends BaseModal {
       type: serverData.metadata.contractType,
       currency: serverData.metadata.pricingCurrency,
       moderated: serverData.moderators && !!serverData.moderators.length,
+      isNew: this.model.isNew(),
```

Comment:
So this will be true the first time the listing is saved, if they continue to tweak things after that, even without closing and re-opening the modal, it would not be new anymore.

That may not change anything here, but just wanted to make that clear.

Reply:
That's exactly what we want here, the intent is to differentiate between the first saves and any further edits.